app/shared/models/build.rb

```diff
@@ -54,38 +51,47 @@ class Build
     # TODO: We currently don't use/store/support parameters (yet) https://github.com/fastlane/ci/issues/783
     attr_accessor :parameters
 
+    # @return [String] contains all information to check out that specific remote, branch, sha, ref again
+    #                  this information will be used when re-running an old build
+    #                  see https://github.com/fastlane/ci/issues/481 for more details
+    attr_accessor :git_fork_config
```

Comment:
Is this mutated outside of where it's set in the constructor? If not, could we make this an `attr_reader` instead?

Reply:
Nice catch